package.json

```diff
@@ -1,115 +1,27 @@
-    "preclean:install": "npm run clean",
-    "preclean:start": "npm run clean",
-    "pree2e": "npm run webdriver:update -- --standalone",
-    "preversion": "npm test",
-    "protractor": "protractor",
-    "reinstall": "npm run clean && npm install",
-    "rimraf": "rimraf",
-    "semantic-release": "semantic-release pre && npm run build:prod && cp -r .git dist && npm publish dist/ && semantic-release post",
-    "server:dev:hmr": "npm run server:dev -- --inline --hot",
-    "server:dev": "webpack-dev-server --config config/webpack.dev.js --progress --profile --watch --content-base src/",
-    "server:dev2:hmr": "npm run server:dev2 -- --hot",
-    "server:dev2": "node config/devserver.js --keepAliveTimeout 10000",
-    "server:prod": "http-server dist --cors",
-    "server": "npm run server:dev",
-    "start:hmr": "npm run server:dev:hmr",
-    "start": "npm run server:dev",
-    "stylelint": "stylelint '**/*.less'; exit 0",
-    "test-old": "npm-run-all --serial test:unit check-coverage test:func",
-    "test": "npm run test:unit && npm run test:func",
-    "test:func": "./run_functional_tests.sh",
-    "test:unit": "karma start",
-    "test:debug": "karma start --no-single-run --browsers Chrome",
-    "test:browsers": "karma start --browsers Chrome,Firefox,Safari",
-    "testserver-start": "webpack-dev-server --inline --progress --host 0.0.0.0 --port 8088",
-    "tslint": "tslint",
-    "typedoc": "typedoc",
-    "version": "npm run build",
-    "watch:dev:hmr": "npm run watch:dev -- --hot",
-    "watch:dev": "npm run build:dev -- --watch",
-    "watch:prod": "npm run build:prod -- --watch",
-    "watch:test": "npm run test:unit  -- --auto-watch --no-single-run",
-    "watch": "npm run watch:dev",
-    "webdriver-manager": "webdriver-manager",
-    "webdriver:start": "npm run webdriver-manager start",
-    "webdriver:update": "npm run webdriver-manager update",
-    "webpack-dev-server": "webpack-dev-server",
-    "webpack": "webpack"
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build --verbose",
+    "prebuild": "mkdir -p node_modules/patternfly/dist/less/bootstrap/less && cp -rf node_modules/bootstrap/less/* node_modules/patternfly/dist/less/bootstrap/less && mkdir -p node_modules/patternfly/dist/less/font-awesome/less && cp node_modules/font-awesome/less/*.less node_modules/patternfly/dist/less/font-awesome/less/",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
   },
+  "private": true,
   "dependencies": {
-    "@angular/animations": "4.4.6",
-    "@angular/common": "4.4.6",
-    "@angular/compiler": "4.4.6",
-    "@angular/compiler-cli": "4.4.6",
-    "@angular/core": "4.4.6",
-    "@angular/forms": "4.4.6",
-    "@angular/http": "4.4.6",
-    "@angular/platform-browser": "4.4.6",
-    "@angular/platform-browser-dynamic": "4.4.6",
-    "@angular/platform-server": "4.4.6",
-    "@angular/router": "4.4.6",
-    "@angular/upgrade": "4.4.6",
+    "@angular/animations": "^6.1.0",
```

Comment:
This is the main application. It can not have ranges for libraries. It needs to be static to be stable.